jcommune-plugin-api/src/main/java/org/jtalks/jcommune/plugin/api/service/PluginTopicDraftService.java

```diff
@@ -0,0 +1,28 @@
+package org.jtalks.jcommune.plugin.api.service;
+
+import org.jtalks.jcommune.model.entity.Topic;
+import org.jtalks.jcommune.model.entity.TopicDraft;
+
+/**
+ * @author Dmitry S. Dolzhenko
+ */
+public interface PluginTopicDraftService {
```

Comment:
I think we can get rid of this interface and use straight `TopicDraftService` instead. Our plugin api is still immature and therefore should be treated with skepticism.